code/modules/core_implant/cruciform/machinery/eotp.dm

```diff
@@ -0,0 +1,196 @@
+var/global/obj/machinery/power/eotp/eotp
+
+#define ARMAMENTS "Armaments"
+#define ALERT "Antag Alert"
+#define INSPIRATION "Inspiration"
+#define ODDITY "Oddity"
+#define STAT_BUFF "Stat Buff"
+#define MATERIAL_REWARD "Materials"
+
+
+/obj/machinery/power/eotp
+	name = "Eye of the Protector"
+	desc = "He observe, he protects."
+	icon = 'icons/obj/eotp.dmi'
+	icon_state = "Eye_of_the_Protector"
+
+	density = TRUE
+	anchored = TRUE
+	layer = 5
```

Comment:
why?

Reply:
So you can go behind it. (It has two vertical layers sprite)

Reply:
Like this.
![image](https://user-images.githubusercontent.com/45293346/102414211-c5efc400-3ffe-11eb-81d5-a23abddf5932.png)